<launch>
    <arg name="node_start_delay" default="5.0" />
    <arg name="port" default="/dev/serial/by-id/usb-RT_CORPORATION_RT-USB-9AXIS-00_9AXIS-00-if00" />
    <arg name="frame_id" default="imu_link" />
    <node pkg="rt_usb_9axisimu_driver" type="rt_usb_9axisimu_driver" name="rt_usb_9axisimu_driver" required="true" output="screen">
        <param name="port" value="$(arg port)" />
        <param name="frame_id" value="$(arg frame_id)" />
    </node>

    <node pkg="imu_utils" type="imu_an" name="imu_an" output="screen" launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' ">
        <param name="imu_topic" type="string" value= "/imu/data_raw"/>
        <param name="imu_name" type="string" value= "rt"/>
        <param name="data_save_path" type="string" value= "/home/sskr3/catkin_ws/src/scanner/data_csv/imu_utils/"/>
        <param name="max_time_min" type="int" value= "120"/>
        <param name="max_cluster" type="int" value= "100"/>
    </node>
</launch>